src/handbook/development/contributing.md

```diff
@@ -16,10 +17,18 @@ If you're using VSCode, then we recommend using the [ESLint extenstion](https://
 
 <img width="429" alt="ESLint - Action on Save" src="../images/eslint_actiononsave.png">
 
-In the case of working with `vue` or `njk` files (found in the [frontend](https://github.com/flowforge/flowforge/tree/main/frontend) and [website](https://github.com/flowforge/website) repositories), then you can add `vue` and `njk` to the `Eslint: Probe` setting in order to enable auto-formatting on save for these file types.
+In the case of working with `vue` or `njk` files (found in the [frontend](https://github.com/flowforge/flowforge/tree/main/frontend) and [website][website-repo] repositories), then you can add `vue` and `njk` to the `Eslint: Probe` setting in order to enable auto-formatting on save for these file types.
 
 <img width="478" alt="ESLint - Probe" src="../images/eslint_probe.png">
 
+#### Editor Config
+
+The [website repository][website-repo] uses a [`.editorconfig`](https://editorconfig.org)
+to allow editors to automatically pick up the correct style for that repository.
+The best editor, neovim, has this pre-installed. If you're using VSCode, an
```

Comment:
Let's bike shed on this for the next 1-2 weeks!

Reply:
Really hoped a line in the middle would do the trick 😄 

```suggestion
Some editors, like neovim, has this pre-installed. If you're using VSCode, an
```